src/devices/legrand.ts

```diff
@@ -717,4 +717,31 @@ export const definitions: DefinitionWithExtend[] = [
             await reporting.onOff(endpoint);
         },
     },
+{
+    zigbeeModel: ['NLIS - Triple light switch'],
+    model: 'NLIS - Triple light switch',
```

Comment:
Please use the model number written on the device page here. (also update the picture pr accordingly)